configs/application/components.json

```diff
@@ -14,26 +14,32 @@
 				"height": 432
 			},
 			"component": {
-				"inject": false,
+				"displayName": "Welcome Component",
 				"spawnOnStartup": false,
+				"inject": false,
 				"preload": "$applicationRoot/preloads/zoom.js"
 			},
 			"foreign": {
 				"services": {
-					"dockingService": {
-						"canGroup": true,
-						"isArrangable": true
+					"windowService": {
+						"allowSnapping": true,
+						"allowGrouping": true,
+						"allowTabbing": true,
+						"allowTiling": true,
+						"allowAutoArrange": true,
+						"allowMinimize": true
```

Comment:
Question. Are these all false by default or are you just putting them in here for illustration?

Reply:
These are true by default and yes included for illustration - this is the config most often copy & pasted so it makes sense to surface some common options (IMHO).